doc/CONTRIBUTING.md

```diff
@@ -12,28 +12,7 @@ and AUTHORS, respectively.
 
 
 
-## Obtaining the source and building the software
-
-This project's source code official repository may be found at 
-
-https://github.com/courselab/ttsnake
-
-If you have obtained the source from the Version Control Repository, you will
-need GNU Build System (aka Autotools) to build the project, as described
-in file `README`, in the root of the project tree (really, read-me). 
-
-Briefly, bootstrap the build system with
-
-```
-$ ./autogen.sh
-```
-
-to create the build configuration script and then build the software with
-
-```
-$ ./configure 
-$ make
-```
+## Obtaining the source and building the software - TODO- Put this somewhere else, see issue 50
```

Comment:
This line can just be removed, since we don't want to leave anything related to installation in this document.

```suggestion
```

Reply:
I think lines 16-31 should be removed as well, don't think it makes much sense for that to be in this particular doc.

Reply:
> I think lines 16-31 should be removed as well, don't think it makes much sense for that to be in this particular doc.

Yeah those are what I meant. Sorry for the confusion. it's 37-53 in the old file and  16-31 in the new one :P
![image](https://user-images.githubusercontent.com/39651883/142096583-5b83d774-5db6-45be-8d40-46f1fce0090c.png)